src/insert.rs

```diff
@@ -256,6 +259,12 @@ impl<T> Insert<T> {
         self.state.terminated();
         self.wait_handle().await
     }
+    
+    /// Returns the number of serialized bytes that have been written because of
+    /// write operations.
+    pub fn written_bytes(&self) -> usize {
```

Comment:
Add `#[inline]`, please, it's supposed to be called on every row, so it's reasonable to allow over-crate inlining.

Also, please, run benchmarks (the `insert` bench with `--features inserter) to ensure we don't introduce regression here.